prisma/schema.prisma

```diff
@@ -1,6 +1,6 @@
 generator client {
   provider      = "prisma-client-js"
-  binaryTargets = ["native", "debian-openssl-1.1.x"]
+  binaryTargets = ["native", "debian-openssl-1.1.x", "debian-openssl-3.0.x"]
```

Comment:
This is the change that fixes Prisma issues. I left the old binary intact in case it's still needed for local development.